manager/src/main/java/com/redhat/service/bridge/manager/api/internal/ShardBridgesSyncAPI.java

```diff
@@ -81,7 +85,9 @@ public Response getProcessors() {
         String shardId = identityResolver.resolve(jwt);
         failIfNotAuthorized(shardId);
         LOGGER.info("Request from Shard for Processors to deploy or delete.");
-        return Response.ok(processorService.getProcessorByStatusesAndShardId(statuses, shardId)
+        List<Processor> processorToDeployOrDelete = processorService.getProcessorByStatusesAndShardId(statuses, shardId);
+        LOGGER.info("Found {} processor(s) to deploy or delete", processorToDeployOrDelete.size());
```

Comment:
If you log this for Processor you might add it also for the Bridge ingresses

Reply:
If we decide to keep this can we also include the `shardId` in the log statement. At the minute it doesn't tell us for which Shard e.g:

`Found '4' processor(s) to deploy or delete for shard 'foo'`